components/feature/addons/src/main/java/mozilla/components/feature/addons/AddonManager.kt

```diff
@@ -70,7 +70,9 @@ class AddonManager(
                 .map { addon -> addon.filterTranslations(locales) }
                 .map { addon ->
                     installedExtensions[addon.id]?.let {
-                        addon.copy(installedState = it.toInstalledState())
+                        val installedState = it.toInstalledState()
+                        addon.copy(installedState = installedState,
+                                version = installedState.version.ifEmpty { addon.version })
```

Comment:
As discussed, I think it's better to keep these separate, the latest add-on version as provided by or AMO collection and the installed version. This will allow us to debug problems, or have telemetry, and also makes it easier to support manual updates later.

Reply:
So, in other words, we don't need to do this and just use the `installedState.version` in the UI if it exists.